pyrk/materials/cool.py

```diff
@@ -0,0 +1,55 @@
+from ur import units
+from density_model import DensityModel
+from material import Material
+import random
+
+class Cool(Material):
```

Comment:
Materials are not meant to be called things like "cool" because many different materials (water, sodium, flinak, flibe, helium) can be a reactor coolant. You seem to be describing flibe. For this, we already have flibe.py. Please delete this file and make necessary changes to flibe.py instead.